src/ckeditor/ckeditor.component.ts

```diff
@@ -115,6 +122,13 @@ export class CKEditorComponent implements AfterViewInit, OnDestroy, ControlValue
 		return this._readOnly;
 	}
 
+	/**
+	 * Fired when the CKEDITOR https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR.html namespace
+	 * is loaded. It only triggers once for each editor component. Can be used for convenient changes
+	 * in the namespace, e.g. for adding external plugins.
+	 */
+	@Output() namespaceLoaded = new EventEmitter<CKEditor4.EventInfo>();
```

Comment:
We went with `namespaceloaded` event name for Vue, it would be good to keep it consistent for easier context switch between integrations.

Reply:
The naming conventions in Vue and Angular are different and we already have the examples in our integrations code (e.g. [`editor-url`](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_vue.html#editor-url) property in Vue vs  [`editorUrl`](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_angular.html#editorurl) property in Angular). Keeping the same name here could be easier for us who develop all the integrations, but for the end-developer using just one of them it will probably look like a bug to see `namespaceloaded` event in Angular 🤔 So my proposal is to keep the naming integration-wise. WDYT?

Reply:
Makes sense, let's keep the `namespaceLoaded` name.